config.def.h

```diff
@@ -146,6 +146,9 @@ static const Key keys[] = {
 
 	/* Ctrl-Alt-Backspace and Ctrl-Alt-Fx used to be handled by X server */
 	{ WLR_MODIFIER_CTRL|WLR_MODIFIER_ALT,XKB_KEY_Terminate_Server, quit, {0} },
+	/* Ctrl-Alt-Fx is used to switch to another VT, if you don't know what a VT is
+	 * Do not remove them
```

Comment:
Not sure if capitalization of "Do" is needed. Also I don't get what does the * in that line do? I guess it's some convention or smth.

Reply:
Ups, it was a typo